samples/net/cellular_modem/boards/nrf52840dk_nrf52840.overlay

```diff
@@ -0,0 +1,27 @@
+/ {
```

Comment:
This does not look right. Now anyone having this DK will get this simcom modem added automatically even if he/she does not have one.

Reply:
I used the nrf52840dk board for testing the modem as none of the supported Zephyr boards use it. Is there a better way to do this? Note that the nrf52840dk board does not have a modem. Or should i just remove it from the sample?